oarepo_cli/less.py

```diff
@@ -27,49 +32,32 @@ class LessComponentsCommands(Commands):
     def register_less_components(self, theme_config_file: Path | None) -> list[Step]:
         """Return steps that register less components."""
         return [
+            FunctionStep(self._collect_less_components, message="Collecting LESS components..."),
             FunctionStep(
-                self._collect_less_components, message="Collecting LESS components..."
-            ),
-            FunctionStep(
-                partial(
-                    self._register_less_components, theme_config_file=theme_config_file
-                ),
+                partial(self._register_less_components, theme_config_file=theme_config_file),
                 message="Registering LESS components...",
             ),
         ]
 
     def _collect_less_components(self) -> ProcessResponse:
         pkg_man = self.cli_config.python_package_manager
         with tempfile.NamedTemporaryFile(mode="w+", suffix=".json") as t:
-            run_cmd(
-                pkg_man.run_command("invenio", "oarepo", "less", "components", t.name)
-            )
+            run_cmd(pkg_man.run_command("invenio", "oarepo", "less", "components", t.name))
             self.components = list(set(json.load(t)["components"]))
         return ProcessResponse()
 
-    def _register_less_components(
-        self, theme_config_file: Path | None
-    ) -> ProcessResponse:
+    def _register_less_components(self, theme_config_file: Path | None) -> ProcessResponse:
         if theme_config_file is None:
             theme_config_file = (
-                self.cli_config.project_path
-                / "ui"
-                / "branding"
-                / "semantic-ui"
-                / "less"
-                / "theme.config"
+                self.cli_config.project_path / ".venv" / "var" / "instance" / "assets" / "less" / "theme.config"
```

Comment:
We should ideally interpoate `INSTANCE_THEME_PATH` config scoped relative to the assets dir: https://github.com/inveniosoftware/cookiecutter-invenio-rdm/blob/3468cec035bfa93fbea9ab295513d2cfaf587daa/%7B%7Bcookiecutter.project_shortname%7D%7D/invenio.cfg#L124. We however don't have access to the app config here, right?